src/controllers/controller.line.js

```diff
@@ -337,7 +337,7 @@ module.exports = function(Chart) {
 
 			model.backgroundColor = custom.hoverBackgroundColor || helpers.valueAtIndexOrDefault(dataset.pointHoverBackgroundColor, index, helpers.getHoverColor(model.backgroundColor));
 			model.borderColor = custom.hoverBorderColor || helpers.valueAtIndexOrDefault(dataset.pointHoverBorderColor, index, helpers.getHoverColor(model.borderColor));
-			model.borderWidth = custom.hoverBorderWidth || helpers.valueAtIndexOrDefault(dataset.pointHoverBorderWidth, index, model.borderWidth);
+			model.borderWidth = custom.hoverBorderWidth || helpers.valueAtIndexOrDefault(dataset.pointHoverBorderWidth, index, this.chart.options.elements.point.hoverBorderWidth);
```

Comment:
What if none of `custom.hoverBorderWidth`, `dataset.pointHoverBorderWidth`, `this.chart.options.elements.point.hoverBorderWidth` are set? Don't we still need to fall back to model.borderWidth in that case?

Reply:
@benmccann in this case can we use a ternary expression to fall back to `model.borderWidth` if both the conditions in `||` are false?

Reply:
as per this line of code https://github.com/chartjs/Chart.js/blob/master/src/helpers/helpers.core.js#L70  I was referring to something like this 
`model.borderWidth = custom.hoverBorderWidth || helpers.valueAtIndexOrDefault(dataset.pointHoverBorderWidth, index, this.chart.options.elements.point.hoverBorderWidth ? this.chart.options.elements.point.hoverBorderWidth : model.borderWidth);`

Reply:
Also, since there are two lines that now refer to `this.chart.options.elements.point` I would refactor that out and create a variable at the beginning of this method with something like: `var pointOpts = this.chart.options.elements.point;`

Reply:
@benmccann I have made the suggested changes in my branch. Can you please let me know if I have missed anything?